src/CoreFoundation/CFBundle.cs

```diff
@@ -581,21 +581,23 @@ public void UnloadExecutable ()
 		}
 
 #if MONOMAC
-#if !NET
+#if NET
+		[SupportedOSPlatform ("macos11.0")]
+		[UnsupportedOSPlatform ("maccatalyst")]
+#else
 		[Introduced (PlatformName.MacOSX, 11, 0)]
 		[NoMacCatalyst]
-#else
-		[SupportedOSPlatform ("macos11.0"), UnsupportedOSPlatform ("maccatalyst15.0")]
 #endif
 		[DllImport (Constants.CoreFoundationLibrary)]
 		[return: MarshalAs (UnmanagedType.I1)]
 		extern static bool CFBundleIsExecutableLoadable (IntPtr bundle);
 
-#if !NET
+#if NET
+		[SupportedOSPlatform ("macos11.0")]
+		[UnsupportedOSPlatform ("maccatalyst")]
```

Comment:
The `[UnsupportedOSPlatform ("ios"), UnsupportedOSPlatform ("tvos")` attributes disasppeared, is that expected?

Reply:
I believe so, CFBundle doesn't have attributes anywhere else FWICT:

```
donblas@Masterwork src % git grep CFBundle | grep interface    
donblas@Masterwork src % git grep CFBundle | grep class        
...
	public partial class CFBundle {
...
donblas@Masterwork src % head -n 25 CoreFoundation/CFBundle.cs 
...
	public partial class CFBundle : NativeObject {
```

and the only legacy attributes are:

```
		[Introduced (PlatformName.MacOSX, 11, 0)]
		[NoMacCatalyst]
```